cypress/integration/BasicSearchPage/BasicSearchPage.feature

```diff
@@ -0,0 +1,219 @@
+    Then the search is not executed and path is "/about-cancer/treatment/clinical-trials/search"
+
+
+Scenario: User is able to search for a trials using age and keyword
+Given screen breakpoint is set to "desktop"
+    Given the user navigates to "/about-cancer/treatment/clinical-trials/search"
+    Then the page title is "Find NCI-Supported Clinical Trials"
+    When user types "breast cancer" in "Cancer Type/Keyword" field
+    And user selects "Breast Cancer" from dropdown
+    And user types "40" in "Age" field
+    And user clicks on "Find Trials" button
+    Then the search is executed and results page is displayed
+		And the criteria table displays the following
+		| Category                      | Selection                 |
+    | Primary Cancer Type/Condition | Breast Cancer             |
+		| Age                           | 40                        |
+    And results info has text "Results 1-10  of 894 for your search "
+    And the url query has the following corresponding code
+      | parameter | value |
+      | loc       | 0     |
+      | rl        | 1     |
+      | t         | C4872 |
+      | a         | 40    |
+
+
+Scenario: User is able to search for all trials
+Given the user navigates to "/about-cancer/treatment/clinical-trials/search"
+    Then the page title is "Find NCI-Supported Clinical Trials"
+    And user clicks on "Find Trials" button
+    Then the search is executed and results page is displayed
+   And trial info displays "Results 1-10  of 6821 for your search for: \"all trials\"   |  Start Over"
+    And the url query has the following corresponding code
+      | parameter | value |
+      | loc       | 0     |
+      | rl        | 1     |
+
+Scenario: User has an option to go to search results url directly and modify search
+Given the user navigates to "/about-cancer/treatment/clinical-trials/search/r?loc=0&rl=1&t=C4872"
+		Then trial info displays "Results 1-10  of 966 for your search "
+    And the criteria table displays the following
+    | Category                      | Selection                                          |
+    | Primary Cancer Type/Condition | Breast Cancer                                      |
+    When user clicks on Modify Search Criteria button
+		And user types "40" in "Age" field
+    And user clicks on "Find Trials" button
+    Then the search is executed and results page is displayed
+		And trial info displays "Results 1-10  of 894 for your search "
+    And the criteria table displays the following
+		| Category                      | Selection                 |
+    | Primary Cancer Type/Condition | Breast Cancer             |
+		| Age                           | 40                        |
+		And the url query has the following corresponding code
+      | parameter | value |
+      | loc       | 0     |
+      | rl        | 2     |
+      | t         | C4872 |
+      | a         | 40    |
+
+
+Scenario: User is able to search for a everything and modify search
```

Comment:
you dont have a kitchen sink scenario - make this a search that incorporates all three fields, e.g. keyword, age and zipcode - and make sure to use type keyword (AIDS, so to make your mocks less painfull) 
so search for AIDS, 30 years and 22182 ( /about-cancer/treatment/clinical-trials/search/r?a=30&loc=1&q=aids&rl=1&z=22182) - and then don't forget to modify it

Reply:
And i will be okay, if you replace the URL of the previous scenario (Scenario: User has an option to go to search results url directly and modify search)  to be a kitchen sink one (/about-cancer/treatment/clinical-trials/search/r?a=30&loc=1&q=aids&rl=1&z=22182) - if you do that, then this scenario can stay as it is